testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/WaitUtils.java

```diff
@@ -48,7 +48,7 @@ public final class WaitUtils {
 
     public static final String PAGELOAD_TIMEOUT_PROP = "pageload.timeout";
 
-    public static final Integer PAGELOAD_TIMEOUT_MILLIS = Integer.parseInt(System.getProperty(PAGELOAD_TIMEOUT_PROP, "10000"));
+    public static final Integer PAGELOAD_TIMEOUT_MILLIS = Integer.parseInt(System.getProperty(PAGELOAD_TIMEOUT_PROP, "60000"));
```

Comment:
This seems to me a bit strange to change this. Increasing this timeout can have more side-effects on other places.

It looks to me that root cause of the issue is somewhere else. Are you able to reproduce this manually?

Reply:
I revisited this. I can't reproduce locally. I did find in the logs that one test from ResetPasswordTest passes before all of the rest fail. Changing the click() calls to clickLink() calls in the Pages used by the passing test seems to fix the timeout issue.